src/EditorFeatures/Core/InlineRename/InlineRenameService.cs

```diff
@@ -85,6 +86,12 @@ internal sealed class InlineRenameService(
             return new InlineRenameSessionInfo(renameInfo.LocalizedErrorMessage);
         }
 
+        var symbolService = document.GetRequiredLanguageService<IGoToDefinitionSymbolService>();
```

Comment:
this is not ok.  you can't ask for a required service that may not exist for all our languages.  TS/JS/Xaml/F# may not implement this.  This has to be GetLanguageService, where you do everything optionally from that point.

Reply:
good to know! fixed